Sources/BottomSheet/Core/Extensions/UIViewController+Convenience.swift

```diff
@@ -95,8 +95,33 @@ public extension UIViewController {
         present(viewController, animated: true, completion: nil)
     }
 
+    func presentBottomSheet(viewController: UIViewController, configuration: BottomSheetConfiguration, dissmisCompletion: @escaping (() -> Void)) {
```

Comment:
Can we make this option optional and nil by default and delete function above `presentBottomSheet(viewController:configuration:)`? Then, we would have much less copy-paste

Reply:
Also, can you please fix a typo?

```Suggestion
    func presentBottomSheet(viewController: UIViewController, configuration: BottomSheetConfiguration, dismissCompletion: @escaping (() -> Void)) {
```